HeroVault/HeroVault/Features/UserProfile/UserProfileViewController.swift

```diff
@@ -9,7 +9,22 @@ import UIKit
 
 class UserProfileViewController: UIViewController {
     
-    override func viewDidLoad() {
-        super.viewDidLoad()
+    // MARK: IBOutlets
+    
+    @IBOutlet weak private var profileLabel: UILabel!
+    @IBOutlet weak private var userImage: UIImageView!
+    @IBOutlet weak private var userNameLabel: UILabel!
+    @IBOutlet weak private var favouriteCharactersLabel: UILabel!
+    @IBOutlet weak private var favouriteCharactersStarImage: UIImageView!
+    @IBOutlet weak private var favouriteComicsLabel: UILabel!
+    @IBOutlet weak private var favouriteComicsStarImage: UIImageView!
+
+// MARK: IBActions
+    
+    @IBAction private func favouriteCharactersButtonTapped(sender: Any) {
+        performSegue(withIdentifier: Constants.SegueIdentifierNames.userFavouritesSegueName, sender: sender)
+    }
+    @IBAction private func favouriteComicsButtonTapped(_ sender: Any) {
+        performSegue(withIdentifier: Constants.SegueIdentifierNames.userFavouritesSegueName, sender: sender)
     }
```

Comment:
If these buttons do the same thing, why not have one button?

Reply:
These buttons will lead to the same page but will populate the table view on the next page with the relevant data.